### ISSUE TITLE
since Sphinx-1.4.4 ``make latexpdf`` generates lots of hyperref warnings

### ISSUE THREAD

**Issue by jfbu** [sphinx-doc/sphinx#2733]
Just try 

``` rst
Welcome to foo's documentation!
===============================

Contents:

.. toctree::
   :maxdepth: 2


AAA

---

aaa

BBB

---

bbb

```

with default `conf.py` (using `manual` docclass for latex output). Then `make latexpdf` generates warnings like those:

```
                      [1]pdfTeX warning (ext4): destination with the same identifier (name{page.2}) has
been already used, duplicate ignored
```

Culprit is an accident in the refactoring of `sphinxmanual.cls` done at PR #2626, which introduced at 9af0583f2 a comment token in 

```
%\pagenumbering{alph}%
```

The problem is that with this `%` the page numbering for the title page and page 2 is the normal one and then as the table of contents resets the numbering  `hyperref` complains about duplicate destinations names due to identical pages.

**Comment by jfbu** [comment]
(the page numbers are not displayed on title page, but internally without `\pagenumbering{alph}` they are arabic digits 1 and then 2)

**Comment by jfbu** [comment]
fixed at 35214d4

sorry for inconvenience for the bug introduced at Sphinx-1.4.4
